### PR TITLE
Added translations support (i18n)

### PR DESCRIPTION
**Pull request by arraintxo**
Hi @DIYgod ,

I added translations support to the extension as that would be useful for us and I saw a pair of issues requesting this feature. (https://github.com/DIYgod/RSSHub-Radar/issues/674 , https://github.com/DIYgod/RSSHub-Radar/issues/33)

Added features:
- Browser language is automatically detected
- Default / Fallback language set to chinese (eventhought, I think english might be a better idea) 
- Supported languages: chinese (default), english, basque and spanish

More languages can be added by creating a language json file for the language and importing it to the i18n.js file

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "access key": "Access key",
+  "about": "About",
+  "click to set": "Click to set",
+  "configuration required if access keys enabled": "Configuration is only required if the instance has access keys enabled, keep it off if you don't know the situation",
+  "copied": "Copied",
+  "copy": "Copy",
+  "current page rss": "RSS on current site",
```
Comment (Rongronggg9) [inline src/translations/en.json:8]:
```suggestion
  "current page rss": "RSS on the current site",
```

File: src/translations/en.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  "current page RSSHub": "RSSHub for the current page",
+  "custom rsshub domain": "Custom RSSHub domain name",
+  "current site RSSHub": "RSSHub for the current site",
+  "documentation": "Info",
+  "enable": "Enable",
+  "enable remote rules": "Enable remote rules",
+  "enter rsshub domain": "Please enter your RSSHub domain name, leave it blank to use the official domain name",
+  "for more rules join us": "For more rules support, <a target=\"_blank\" href=\"https://docs.rsshub.app/joinus/\">join us</a>！",
```
Comment (Rongronggg9) [inline src/translations/en.json:16]:
```suggestion
  "for more rules join us": "For more rules support, <a target=\"_blank\" href=\"https://docs.rsshub.app/en/joinus/quick-start.html\">join us</a>！",
```

File: src/translations/en.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  "general": "General",
+  "generate access code": "Generate access code",
+  "hot key": "Hot key",
+  "join rsshub": "Participate in the <a href=\"https://docs.rsshub.app\"> to make everything RSS-able!",
```
Comment (Rongronggg9) [inline src/translations/en.json:20]:
```suggestion
  "join rsshub": "Participate in the <a href=\"https://docs.rsshub.app/en/\"> to make everything RSS-able!",
```

File: src/translations/es.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  "copy": "Copiar",
+  "current page rss": "RSS en esta página",
+  "current page RSSHub": "RSSHub para página actual",
+  "custom rsshub domain": "Dominio RSSHub personalizado",
+  "current site RSSHub": "RSSHub para website actual",
+  "documentation": "Info",
+  "enable": "Activar",
+  "enable remote rules": "Habilitar reglas remotas",
+  "enter rsshub domain": "Introduce to dominio de RSSHub, deja el espacio en blanco para utilizar el dominio oficial",
+  "for more rules join us": "Para más reglas <a target=\"_blank\" href=\"https://docs.rsshub.app/joinus/\">únete</a>！",
```
Comment (Rongronggg9) [inline src/translations/es.json:16]:
```suggestion
  "for more rules join us": "Para más reglas <a target=\"_blank\" href=\"https://docs.rsshub.app/en/joinus/quick-start.html\">únete</a>！",
```

File: src/translations/es.json [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  "general": "General",
+  "generate access code": "Generar código de acceso",
+  "hot key": "Hot key",
+  "join rsshub": "Únete a <a href=\"https://docs.rsshub.app\">RSSHub</a> para hacer todo accesible mediante RSS!",
```
Comment (Rongronggg9) [inline src/translations/es.json:20]:
```suggestion
  "join rsshub": "Únete a <a href=\"https://docs.rsshub.app/en/\">RSSHub</a> para hacer todo accesible mediante RSS!",
```

File: src/js/common/i18n.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+import eu from '../../translations/eu.json';
+import zh from '../../translations/zh.json';
+const i18n = new I18n({
+    en,
+    es,
+    eu,
+    zh,
+});
+i18n.enableFallback = true;
+i18n.defaultLocale = 'zh';
```
Comment (Rongronggg9) [inline src/js/common/i18n.js:13]: Why not `en`?

File: src/translations/en.json [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+  "settings": "Settings",
+  "show corner badge": "Show corner badge",
+  "subscribe to": "Subscribe to",
+  "subscribe to local reader": "Subscribe to local reader",
+  "successfully saved": "Successfully saved",
+  "updated time ago": "%{hours}h %{minutes}m since last update",
+  "version": "Version",
+
+  "rsshub radar info": "RSSHub Radar is a spin-off of <a target=\"_blank\" href=\"https://docs.rsshub.app/en/\">RSSHub</a> that helps you quickly discover and subscribe to RSS and RSSHub for your current site, and the project is <a target=\"_blank\" href=\"https://github.com/DIYgod/RSSHub-Radar\">open source</a> under the MIT license and is completely free to use, but as the project grows in size, it also needs to be funded to sustain its maintenance and development",
+  "sponsored development": "Sponsored development of RSSHub Radar: <a target=\"_blank\" href=\"https://docs.rsshub.app/en/support\">https://docs.rsshub.app/en/support</a>",
```
Comment (Rongronggg9) [inline src/translations/en.json:41]:
```suggestion
  "sponsored development": "Sponsored development of RSSHub Radar: <a target=\"_blank\" href=\"https://docs.rsshub.app/en/support/\">https://docs.rsshub.app/en/support/</a>",
```

File: src/translations/es.json [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+  "settings": "Opciones",
+  "show corner badge": "Mostrar insignia",
+  "subscribe to": "Subscribirse a",
+  "subscribe to local reader": "Subscribirse en lector local",
+  "successfully saved": "Guardado",
+  "updated time ago": "%{hours}h %{minutes}m desde la última actualización",
+  "version": "Versión",
+
+  "rsshub radar info": "RSSHub Radar es un proyecto derivado de <a target=\"_blank\" href=\"https://docs.rsshub.app/en/\">RSSHub</a> que permite subscribrirse rápidamente al RSS y/o RSSHub del website actual. Es un proyecto <a target=\"_blank\" href=\"https://github.com/DIYgod/RSSHub-Radar\">open source</a> bajo licencia MIT y su uso es completamete libre y gratuito, pero a medida que el proyecto crece también necesita ser patrocinado para mantener su soporte y desarrollo",
+  "sponsored development": "Soporta el desarrollo de RSSHub Radar: <a target=\"_blank\" href=\"https://docs.rsshub.app/en/support\">https://docs.rsshub.app/en/support</a>",
```
Comment (Rongronggg9) [inline src/translations/es.json:41]:
```suggestion
  "sponsored development": "Soporta el desarrollo de RSSHub Radar: <a target=\"_blank\" href=\"https://docs.rsshub.app/en/support/\">https://docs.rsshub.app/en/support/</a>",
```

File: src/translations/eu.json [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+  "copy": "Kopiatu",
+  "current page rss": "Orrialde honetako RSSak",
+  "current page RSSHub": "Orrialde honetarako RSSHub-ak",
+  "custom rsshub domain": "RSSHub domeinu pertsonalizatua",
+  "current site RSSHub": "Gune honetarako RSSHub-ak",
+  "documentation": "Info",
+  "enable": "Gaitu",
+  "enable remote rules": "Hurruneko arauak gaitu",
+  "enter rsshub domain": "Mesedez sartu zure RSSHub domeinua, domeinu ofiziala erabiltzeko utzi zuriz",
+  "for more rules join us": "Arau gehiago jasatzeko <a target=\"_blank\" href=\"https://docs.rsshub.app/joinus/\">batu zaitez</a>!",
```
Comment (Rongronggg9) [inline src/translations/eu.json:16]:
```suggestion
  "for more rules join us": "Arau gehiago jasatzeko <a target=\"_blank\" href=\"https://docs.rsshub.app/en/joinus/quick-start.html\">batu zaitez</a>!",
```

File: src/translations/eu.json [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+  "general": "Orokorra",
+  "generate access code": "Sarrera kodea sortu",
+  "hot key": "Hot key",
+  "join rsshub": "Gehitu zure burua <a href=\"https://docs.rsshub.app\">RSSHub</a>-era dena RSS bitartez jaso ahal izateko!",
```
Comment (Rongronggg9) [inline src/translations/eu.json:20]:
```suggestion
  "join rsshub": "Gehitu zure burua <a href=\"https://docs.rsshub.app/en/\">RSSHub</a>-era dena RSS bitartez jaso ahal izateko!",
```

File: src/translations/eu.json [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+  "settings": "Aukerak",
+  "show corner badge": "Izkinako marka erakutsi",
+  "subscribe to": "Harpidetu: ",
+  "subscribe to local reader": "Harpidetu irakurgailu lokalean",
+  "successfully saved": "Gordeta",
+  "updated time ago": "%{hours}o %{minutes}m azken eguneraketatik",
+  "version": "Bertsioa",
+
+  "rsshub radar info": "RSSHub Radar momentuko webgunearen RSS eta RSSHub-era modu azkar eta erraz batean harpidetzera laguntzen duen <a target=\"_blank\" href=\"https://docs.rsshub.app/en/\">RSSHub</a>-en proiektu eratorria da. Guztiz erabilera librea du eta MIT lizentziapean dagoen <a target=\"_blank\" href=\"https://github.com/DIYgod/RSSHub-Radar\">kode irekiko</a> proiektua da, baina handitzen doan heinean bere mantentze eta garapenari eusteko finantzazio beharra dauka.",
+  "sponsored development": "RSSHub Radar laguntzeko: <a target=\"_blank\" href=\"https://docs.rsshub.app/en/support\">https://docs.rsshub.app/en/support</a>",
```
Comment (Rongronggg9) [inline src/translations/eu.json:41]:
```suggestion
  "sponsored development": "RSSHub Radar laguntzeko: <a target=\"_blank\" href=\"https://docs.rsshub.app/en/support/\">https://docs.rsshub.app/en/support/</a>",
```

File: src/translations/zh.json [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+  "copy": "复制",
+  "current page rss": "当前页面上的 RSS",
+  "current page RSSHub": "适用于当前页面的 RSSHub",
+  "custom rsshub domain": "自定义 RSSHub 域名",
+  "current site RSSHub": "适用于当前网站的 RSSHub",
+  "documentation": "文档",
+  "enable": "开启",
+  "enable remote rules": "远程更新被禁用",
+  "enter rsshub domain": "请输入你的 RSSHub 域名，留空使用官方域名",
+  "for more rules join us": "更多规则支持中，快来<a target=\"_blank\" href=\"https://docs.rsshub.app/joinus/\">参与我们</a>吧！",
```
Comment (Rongronggg9) [inline src/translations/zh.json:16]:
```suggestion
  "for more rules join us": "更多规则支持中，快来<a target=\"_blank\" href=\"https://docs.rsshub.app/joinus/quick-start.html\">参与我们</a>吧！",
```

**Review by Rongronggg9 — CHANGES_REQUESTED** [review]

File: src/js/common/i18n.js (hunk 6 above)
Comment (arraintxo) [reply src/js/common/i18n.js]: I set `en` as default language

**Pull request merged**